src/main/java/com/minecolonies/coremod/inventory/GuiMinecoloniesCitizenInventory.java

```diff
@@ -27,8 +27,6 @@ public GuiMinecoloniesCitizenInventory(final ContainerMinecoloniesCitizenInvento
         this.upperChestInventory = containerMinecoloniesBuildingInventory.getPlayerInventory();
         this.lowerChestInventory = containerMinecoloniesBuildingInventory.getLowerChestInventory();
         this.allowUserInput = false;
-        final int i = 222;
-        final int j = 114;
         this.inventoryRows = containerMinecoloniesBuildingInventory.getLowerChestInventory().getSizeInventory() / 9;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 9 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)